src/api/routes/policies/stats.ts

```diff
@@ -0,0 +1,18 @@
+import { Router, Request, Response } from "express";
+import { route } from "@fosscord/api";
+import { Config, User, UserPublic, Member, Guild, Message } from "@fosscord/util";
+const router = Router();
+
+
+router.get("/",route({right:"VIEW_SERVER_STATS"}), async (req: Request, res: Response) => {
```

Comment:
IMO stats requiring authorization should be optional, as it brings up some interesting use cases, like instance lists showing user count etc.